sdk/entra/ci.yml

```
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/entra/WebJobs.Extensions.AuthenticationEvents/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/entra/WebJobs.Extensions.AuthenticationEvents/
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: template
```

This should match your service directory otherwise you are testing our template project.

```suggestion
    ServiceDirectory: entra
```

---

Done. And Thank you...